_data/events_gdcr2019/Paris.json

```diff
@@ -0,0 +1,32 @@
+{
+  "title": "GDCR19 Paris",
+  "url": "",
```

Comment:
Thanks @iAmDorra :tada: Please, build check failed because there isn't an url for the event page (Eventbrite, Meetup, ...). We'll keep this pull request open so it's only necessary to update the file, commit and push on your branch again. Pull request will be updated automatically.

Reply:
Ok, i'll create the event and update the file as soon as possible ;)
Tks